### ISSUE TITLE
Wiring question and hurray!

### ISSUE THREAD

**Issue by fondberg** [bruxy70/Irrigation-with-display#1]
First off! Really cool project. Read thqt you want to get into HA more. Maybe i can help. I have a custom card for lovelace and a couple of components and work activelly with HA every day.

Anyhow, i got my sonoff and my nextion display (NX4024K032) but the pins are not labelled kn the nextion. Could you share how it is wired? 
My screen got damaged in the mail but it it looks like this

![1599048362542936602703688731541](https://user-images.githubusercontent.com/10739838/91979069-7aeac300-ed25-11ea-8edb-367b8fad5e07.jpg)
The front has nothing except the tft.

Did i buy the wrong display?

**Comment by bruxy70** [comment]
It is a different display to what I use.
Check out this video: https://youtu.be/_u1sOCfyghY

**Comment by fondberg** [comment]
Bummer. Thanks for the link. 
Did you dremel out the hole for the display in the sonoff chassi?

**Comment by bruxy70** [comment]
Yes, I did. That's another video :) https://youtu.be/MyaxiW9ofXc

**Comment by fondberg** [comment]
Great! 
Brilliant videos!
Looking at my sonoff I bought the wrong one too :-/ Mine is AC in... 
Back to shopping guess.
